client-src/elements/form-field-specs.js

```diff
@@ -377,6 +398,14 @@ export const ALL_FIELDS = {
         Link to the spec, if and when available. When implementing
         a spec update, please link to a heading in a published spec
         rather than a pull request when possible.`,
+    extra_help: html`
+        <p>
+        In addition to starting or updating your specification,
+        start sketching out a proposed solution in your (public) explainer
+        during your prototyping, detailing the API design (using webIDL).
+        It's also a good idea to discuss your idea with the team/team lead (TL)
+        or area expert in the feature area, prior to checking in code in the area.
+        </p>`,
```

Comment:
I think this doesn't really fit as extra_help for the spec field, since it happens long before most teams should actually start writing the specification. We might need to just drop this text for this PR, but @jrobbins, what do you suggest for help text that belongs on the phase rather than any particular field?

Reply:
I agree that this is too high-level for field-specific extra-help.

We are moving toward a more approval-centric approach with more approval gates owned by more teams, even if some of the gates are auto-approved when the feature owner fills in all fields.  As part of that UX change, there is a new location available for process help text.  We call it the "questionnaire", but it can be a paragraph of any text, not neccessarily questions.   Is there some team that would logically be the ones giving the advice proposed in this part of the PR? Adoption, Interop, Security, Privacy, UX, A11y, Testing, Debuggability, DevRel, Enterprise, or overall API owners?

Alternatively, I suppose that we could add support for extra help on the stages themselves.

Reply:
I think this is an "overall API owners" thing, or at least, none of the other areas in your list seem right. The core text comes from https://www.chromium.org/blink/launching-features/#step-1-incubating-write-up-use-cases-and-scenarios-in-an-explainer, which is from the API owners.

Reply:
I'll leave this extra_help out for now.  I like the idea of the help text (and maybe extra_help as well) for the stages.  

While each type of approver should perhaps be able to get a view of the data that is specifically only what they are interested in, the users who are making changes to the data regarding a particular stage might be interested in everything that needs to be done for that stage.  If those users should not have to look under half a dozen other views (for each approval process) to find all the info they need.